config.js

```diff
@@ -1,10 +1,13 @@
 /* eslint import/extensions: 0 */
 import packageJson from "./package.json";
 
-export default () => ({
-  libraryRepoUrl: packageJson.repository.url,
-  demoRepoUrl: packageJson.repository.url,
-  copyright: "Copyright 2021 Inrupt, Inc.",
-  demoTitle: "SOAP",
-  demoDescription: "A Solid ODRL Access Policies editor",
-});
+export default function getConfig() {
+  return {
+    defaultIdP: "https://broker.pod.inrupt.com",
+    libraryRepoUrl: packageJson.repository.url,
+    demoRepoUrl: packageJson.repository.url,
+    copyright: "Copyright 2021 Inrupt, Inc.",
+    demoTitle: "SOAP",
```

Comment:
```suggestion
    copyright: "Copyright 2022 Beatriz Esteves",
    demoTitle: "SOPE",
```

Reply:
I'll leave that to @besteves4

Reply:
Though these values aren't currently used anywhere on the whole, only the demoTitle and defaultIdP values are used at present.